ExampleFhirClient/CertificateStore.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+
+namespace ExampleFhirClient
+{
+    class CertificateStore
+    {
+	    public X509Certificate2 GetMostRecentDistinguishedNameCertificate( StoreName storeName, StoreLocation storeLocation, string distinguishedName )
```

Comment:
We have only one certificate - and if it is expired nothing will work in any case - so maybe the code here can be simplified (in the interest of more clarity - we are not really trying to exemplify how to manage certificates)

Reply:
I've greatly simplified the cert access as you've suggested.